crates/pet-windows-store/src/environments.rs

```diff
@@ -63,14 +63,15 @@ impl PotentialPython {
                 } else {
                     None
                 })
-                .version(Some(self.version.clone()))
+                // We only have the partial version, no point returning bogus info.
+                // .version(Some(self.version.clone()))
```

Comment:
Another change to never return bogus information (partial version != valid data)
And Pet does best effort to get valid/accurate data, or nothing at all.

Reply:
@karthiknadig /cc